packages/react-frontend/src/routes/AlbumInfo.jsx

```diff
@@ -37,9 +54,35 @@ export default function AlbumInfo() {
       });
   }
 
+  function calculateAverage(reviews){
+      if(reviews.length === 0) return 0;
+      // .reduce , iterates through array of reviews and sums up all ratings
+      const totalRating = reviews.reduce((sum, review)=> sum + review.rating, 0);
+      return totalRating / reviews.length;
+  }
+
+  function getStarRating(rating){
+    
+    const fullStars = Math.floor(rating);
+    const halfStar = rating % 1 >= 0.5 ? 1: 0;
+    const emptyStars = 5 - (fullStars + halfStar)
+    // half star is just going to be empty for now
+    return '★'.repeat(fullStars) + (halfStar ? '★' : '') + '☆'.repeat(emptyStars);
```

Comment:
Is the half star showing up hard to implement? It'd definitely help users understand the rating system better and it'll look nicer